olive/common/config_utils.py

```diff
@@ -115,6 +115,41 @@ def from_json(cls, json_dict: dict) -> "ConfigBase":
         return cls.parse_raw(json.dumps(json_dict))
 
 
+class ConfigListBase(ConfigBase):
```

Comment:
what's the consideration that we don't use the native list/dict and use the customized list/dict?

Reply:
Use to simplify the access on pydantic data. Not aim to customized list/dict.